Backend/app/dashboard/CommonAPIs/spec/ProviderPlatform/Fleet/API/Onboarding.yaml

```
@@ -0,0 +1,58 @@
+    isMandatory: Bool
+    isDisabled: Bool
+    disableWarning: Maybe Text
+    isHidden: Bool
+    dependencyDocumentType: '[DocumentType]'
+    checkExtraction: Bool
+    checkExpiry: Bool
+    rcNumberPrefixList: '[Text]'
+    filterForOldApks: Maybe Bool
+    documentCategory: Maybe DocumentCategory
+  DocumentCategory:
+      enum: "Driver,Vehicle,Permission,Training"
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)